Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Dockerfile for running tests only.
+
+
+FROM python:3.6-slim
```

Comment:
I believe we should be moving toward python3.7

Reply:
There's still a 3.6 environment that I want to use this library in, and so far there aren't any 3.7 features I want to use in the library. Do you think it's worth running the tests in both versions?

Reply:
As discussed offline, I'll merge this as-is.